_api_app/app/Sites/SitesDataService.php

```diff
@@ -63,6 +63,7 @@ class SitesDataService extends Storage
     private $ROOT_ELEMENT = 'sites';
     private $SITES = array();
     private $XML_FILE;
+    private $MAIN_SITE_DEFAULT_TITLE = "Main site";
```

Comment:
please use single quotes `'` for string definition unless  you need to format string `"some string $someVariable"` or you to use quotes in the string `"it's a string"`